cli/test/test_api_wrapper.py

```
@@ -0,0 +1,245 @@
+
+    def mock_get(*args, **kwargs):
+        return MockGetListOfProjectsResponse()
+
+    monkeypatch.setattr(requests, "get", mock_get)
+
+    if should_raise:
+        with pytest.raises(Exception):
+            api_wrapper.get_list_of_projects()
+
+    else:
+        api_wrapper.get_list_of_projects() == expected
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)